Fabric.Authorization.API/scripts/Install-Authorization-Utilities.psm1

```diff
@@ -270,6 +270,43 @@ function Add-RoleToGroup
     Invoke-Sql $connString $query @{groupId = $groupId; roleId = $roleId; ; clientId = $clientId} | Out-Null
 }
 
+function Add-UserOrGroupToEdwAdmin($userOrGroup, $connString) {
+	$query   = "DECLARE @userId INTEGER;
+                DECLARE @roleId INTEGER;
+                
+                SET @userId = -1
+                SELECT @userId = COALESCE([IdentityID], -1) FROM [EDWAdmin].[CatalystAdmin].[IdentityBASE] WHERE [IdentityNM] = @identityName
```

Comment:
this is a global variable, right?

Reply:
That is a sql parameter variable.

Reply:
I'm not seeing where it gets it's value set